### DISCUSSION TITLE
PCE Controls

### DISCUSSION THREAD

**Discussion by pmgducati** [darthcloud/BlueRetro#167] [Q&A] [answered]
@darthcloud, thanks again for the help in my other discussion with the buttons/HW issue. (https://github.com/darthcloud/BlueRetro/discussions/165).  I opened this discussion as the root problem (double presses) in the previous discussion was resolved.

I have been testing with the 6 button setup and all two button games freak out if alt mode is activated (This could be related to the random presses you were asking about in the previous conversation). This is not an issue with your firmware, rather it is a issue with the original hardware, this would happen with the original 6 button controller.   This is why there was a 2 button/6 button switch on the controller. As I mentioned in the previous discussion the two button version layout is not playable with Button I on the right trigger. With that said I looked into the 6 button controller game list and it is VERY short (see Below) in the large list of PCE/TG16 games. 

I am able to fix this by remapping the buttons, but it then breaks the config for the other systems that I am using this for (As of now Gamecube, 3DO, NES with 3-4 additional in the works).  Would it be possible to create a profile with II and I on the A/B Buttons and a second for the 6 button controller? 

6 Button Game List:
Advanced VG
Art of Fighting
Battlefield '94
Fatal Fury 2
Fatal Fury Special
Fire Pro Female Wrestling
Kabuki Itouryodan
Kakutou Haou Densetsu Algunos
Linda Cube
Majhong Sword Princess Quest Gaiden
Martial Champions
Princess Maker 2
Street Fighter II'
Strip Fighter
Super Real Mahjong P ll + lll Custom
World Heroes 2
Ys IV

**Comment by darthcloud** [comment, marked as answer]
Yes I will unlink the gamepad & GamePadAlt default mapping, so this way 2 btns mode will have I on right face, and II on bottom face, and select/III on left face. Probably in v1.3

**Reply by pmgducati** [reply]
That is Awesome!  I will keep my eyes open for it.  If you need anyone to test it feel free to reach out!
